iothrottler.go

```diff
@@ -69,7 +69,7 @@ func throttlerPoolDriver(pool *IOThrottlerPool) {
 	currentBandwidth := Bandwidth(0)
 	totalbandwidth := Bandwidth(0)
 	allocationSize := Bandwidth(0)
-	var timeout <-chan time.Time = nil
+	timeout := time.NewTicker(time.Second)
```

Comment:
Thanks for posting this. I'm happy about the fix to the leak, but I'm concerned that this is starting the timer at a different time than the original code. In the original code we don't start the time until we get our first client. This won't matter for many workloads, but I think it will break other workloads. Could you change this so that the timer doesn't start until we get the first client?

Reply:
done:)